corehq/apps/sms/tests/migration.py

```diff
@@ -8,13 +12,16 @@ class BaseMigrationTestCase(TestCase):
 
     def setUp(self):
         self.domain = 'test-sms-sql-migration'
-        self.deleteAllLogs()
+        self.deleteAllObjects()
```

Comment:
nit: It's always weird to me that we use this pattern. deleting objects in the setup seems like there are issues in other tests teardown that should be fixed?